### PR TITLE
Lung cancer intervention

### PR DESCRIPTION
**Pull request by dehall**
Updates the Lung Cancer module to take into account lung cancer screening interventions.
CHW encounter performs a Low-dost CT scan if they offer the service and the patient meets the necessary criteria.

The Lung Cancer module is updated so that the screening will affect the stage of cancer the person has when they start treatment. This is accomplished by adding a counter to the start of the module, counting the duration of how long the person has undetected lung cancer. This loop can either be broken out of randomly, or if they get the screening provided by a CHW.

The numbers will likely need to be tweaked but this is a first pass.

Updated module image:
lung cancer

### CODE REVIEW

**Comment by dehall** [comment]
@jawalonoski To simulate a "referral" should I create a new encounter at some random point in the future? Or is this good enough for now? At some point I would like to add the concept of referrals to the system, but I think that's a ways off.

@Abdulrahims I'd leave the CVD stuff in its own module. Lifecycle is getting fairly big and may get hard to follow if it contains a lot of different things.

**Comment by jawalonoski** [comment]
Yes, ideally we'd create a referral and that would create a new encounter at a later data with an appropriate provider who would provide the service. But I don't want to do that now. As my comment indicated, I like it this way for the time being.
